packages/@aws-cdk/aws-appmesh/lib/virtual-service.ts

```diff
@@ -211,3 +179,68 @@ export interface VirtualServiceAttributes {
    */
   readonly clientPolicy?: ClientPolicy;
 }
+
+/**
+ * Properties for a VirtualService provider
+ */
+export interface VirtualServiceProviderConfig {
+  /**
+   * Virtual Node based provider
+   *
+   * @default - none
+   */
+  readonly virtualNodeProvider?: CfnVirtualService.VirtualNodeServiceProviderProperty;
+
+  /**
+   * Virtual Router based provider
+   *
+   * @default - none
+   */
+  readonly virtualRouterProvider?: CfnVirtualService.VirtualRouterServiceProviderProperty;
```

Comment:
Let's add a third, required, property here, `mesh: IMesh`.

Reply:
This is present. May be a stale comment?